content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -0,0 +1,106 @@
+
+To draft and submit a blog post, you will need:
+
+* A GitHub account known to O3DE and a basic understanding of simple Git workflows
+* A working knowledge of Markdown syntax
+
+If you have never participated in our communities before, we highly recommend you spend some time socializing in them and joining at least one SIG meeting before you start blogging. Regardless, blogs will be reviewed by community maintainers before they are published.
+
+## How the O3DE blog works
+
+It's just like the [docs contribution workflow](get-started.md).
+
+1. Fork the https://github.com/o3de/o3de.org repo.
+
+2. Clone your fork to your local machine. Set `upstream` to o3de.org and `origin` to your fork.
+
+3. Create a branch (preferably one specific to JUST your blog post) for submission.
+
+4. Open your editor of choice and create a draft post using Markdown under thr `/content/blog/posts` folder. Give it a clear, unique filename in all lower-case, with no spaces. (Please use a hyphen `-` as a replacement for spaces. Avoid all other non-alphanumeric characters.)
+
+5. Make sure that your blog post has the following metadata at the top:
+
+    ```metadata
+    ---
+    title: "YOUR TITLE IN QUOTES HERE. KEEP IT LESS THAN 80 CHARACTERS."
+    date: YYYY-MM-DD
+    slug: UNIQUE STRING FOR YOUR POST
+    author: YOUR PREFERRED AUTHOR NAME HERE. EMPLOYER IS OPTIONAL!
+    blog_img: "/images/blog/YOUR-OPTIONAL-ASSOCIATED-IMAGE-THUMBNAIL-HERE"
+    full_img: "/images/blog/  YOUR-OPTIONAL-ASSOCIATED-FULL-IMAGE-HERE"
+    ---
+    ```
+
+    As an example:
+
+    ```metadata
+    ---
+    title: "Announcing Open 3D Engine!"
+    date: 2021-07-06
+    slug: welcome-post
+    author: Doug Erickson, Amazon Web Services
+    blog_img: "/images/blog/announcement_thumbnail.jpg"
+    full_img: "/images/blog/atom_showcase.png"
+    ---
+    ```
+
+    Note that the blog is automated and that the `date` field determines when the blog will first become visible if your pull request is accepted and merged.
+6. Commit your changes to your branch. **Ensure your commit(s) are DCO signed!***
+
+7. Push your branch to your fork (`origin`). Go to your fork on GitHub and create a pull request from your fork/branch to o3de/o3de.org/main (`upstream`).
+
+8. Go to your pull request at https://github.com/o3de/o3de.org/pulls and ensure that all checks have passed. If they haven't, fix up your pull request and resubmit it.
+
+9. Go into the O3DE Discord server and share your pull request for review. (Alternatively, mail the sig-docs-community mailing list with a link.) Someone will review it and provide feedback or an okay. A maintainer will merge it when at least one reviewer approves it.
```

Comment:
Is there a process that is documented about expected turnaround time for review, or a place we might want to include the approval process in the future? Not sure if it's for here or the charter (or some other document).